Wireframe/index.html

```diff
@@ -3,31 +3,62 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wireframe</title>
+    <title>Wireframe project</title>
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
+      <h1>Wireframe Project</h1>
+      <p>About README file,Wireframe, and Git Branches</p>
     </header>
+
     <main>
+      <section class="articles">
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img src="icons8-info.svg" alt="README file illustration" class="article-image">
+        <h2>README file</h2>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+          A <str>README</str> file is used to provide essential infomation about a Project:
         </p>
-        <a href="">Read more</a>
+        <ol>
+          <li>Explaining what the project is about </li>
+          <li>Giving instruction on how to install,use,or contribute to the project</li>
+          <li>Listing prerequisites,technology used,and authors</li>
+          <li>Acting as the first point of reference for anyone viweing the project</li>
+        </ol>
+        <a href="https://shorturl.at/IyMPe" class="read-more" >Read more</a>
       </article>
-    </main>
-    <footer>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </footer>
+
+
+      <article>
+      <img src="wireframe.svg" alt="wireframe illustration" class="article-image">
```

Comment:
Hi @RichSwiftie  do not forget to end your image tag with /> because it is an image tag one do not have to have a closing tag, please check the example and how I close the images <img src="wireframe.svg" alt="wireframe illustration" class="article-image" />